charts/rancher-monitoring/v0.1.3/charts/operator-init/files/crd-prometheusrules.yaml

```diff
@@ -1,10 +1,9 @@
-# https://raw.githubusercontent.com/coreos/prometheus-operator/release-0.38/example/prometheus-operator-crd/monitoring.coreos.com_prometheusrules.yaml
-apiVersion: apiextensions.k8s.io/v1beta1
+# https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/release-0.39/bundle.yaml
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
They switched from `apiextensions.k8s.io/v1beta` to `apiextensions.k8s.io/v1` as part of https://github.com/prometheus-operator/prometheus-operator/blame/99ecd7100c2dd9c1392e7670215045cf41606a0f/CHANGELOG.md#L70.